erpnext/hr/report/employee_leave_balance/employee_leave_balance.py

```diff
@@ -39,19 +39,22 @@ def get_data(filters, leave_types):
 	
 	data = []
 	for employee in active_employees:
-		row = [employee.name, employee.employee_name, employee.department]
-
-		for leave_type in leave_types:	
-			# leaves taken
-			leaves_taken = get_approved_leaves_for_period(employee.name, leave_type, 
-				filters.from_date, filters.to_date)
+		leave_approvers = [l.leave_approver for l in frappe.db.sql("""select leave_approver from `tabEmployee Leave Approver` where parent = %s""",
+							(employee.name),as_dict=True)]
+		if (len(leave_approvers) and user in leave_approvers) or (user=="Administrator") or ("HR Manager" in frappe.get_roles(user)):
```

Comment:
Employee should also be able to see his/her leave balance.

`user in ["Administrator", employee.user_id]`